test/kubernetes.spec.mts

```diff
@@ -202,6 +204,57 @@ describe('generator - Kubernetes', () => {
     it('creates expected gateway files', () => {
       assert.file(expectedFiles.jhgate);
     });
+    it('creates expected gateway ingress files', () => {
+      assert.file(expectedFiles.jhgateingress);
+    });
+    it('create the apply script', () => {
+      assert.file(expectedFiles.applyScript);
+    });
+  });
+
+  describe('gateway and ingressType gke', () => {
+    let runResult;
+    before(async () => {
+      runResult = await helpers
+        .createJHipster(GENERATOR_KUBERNETES)
+        .doInDir(dir => {
+          createMockedConfig('01-gateway', dir, { appDir: '01-gateway', config: { authenticationType: 'oauth2' } });
+        })
+        .withOptions({ skipChecks: true, reproducibleTests: true })
```

Comment:
Deployment tests have been refactored at at main.
Updated implementation will be:
```suggestion
    const chosenApps = ['01-gateway'];
    before(async () => {
      runResult = await helpers
        .generateDeploymentWorkspaces({ authenticationType: 'oauth2' })
        .withWorkspacesSamples(...chosenApps)
        .withGenerateWorkspaceApplications();

      runResult = await runResult
        .create(getGenerator(GENERATOR_KUBERNETES))
```

Reply:
Thanks for the notice. I have to re-work the PR because Consul was merged as well.